### ISSUE TITLE
Froala Editor v4 Support

### ISSUE THREAD

**Issue by Panman82** [froala/ember-froala-editor#104]
Plan is to release an alpha in the very near term that enables Froala Editor v4 support with this addon in the given format. Then work on the Embroider and ember-auto-import upgrades before releasing the v4.0 of this addon.

**Comment by tavosansal** [comment]
Hey @Panman82 ... thank you so much for all of your work on this. I know for v3 you were able to push out a version that was compatible with the new editor without having to change the addon. Do you think this would be possible for v4?
v3 has some bugs that were address on v4 and we were wondering if this is something we could update to soon. 

Thank you.

**Comment by Panman82** [comment]
Yes, sorry for the delay, I've had a major work project that has been my focus. I'll set some time aside next week to look closer at this. The v4 upgrade should be easy, but there are ember addon ecosystem changes that would be nice to include in a v4 release here.

**Comment by tavosansal** [comment]
thanks @Panman82

**Comment by Panman82** [comment]
So sorry for the delay here, I'll be working on this over the next few weeks. Looks like there are some big changes with addons for Embroider and such that should be incorporated for 4.0 of this addon.

**Comment by tavosansal** [comment]
@Panman82 is there a way I could buy you a beer or a coffee for doing this?

**Comment by Panman82** [comment]
> @Panman82 is there a way I could buy you a beer or a coffee for doing this?

Nope, this is my contribution to the ember community. As a fellow ember dev, and user of the Froala Editor, I would be doing this work one way or another. 😅 

RE: Release; I'm trying to bring in embroider compatibility so that v4 here will jive with ember v4 without issue. However, that might impact how the Froala assets are imported (and how to configure what plugins/lang/themes are imported), so I need to get it in before releasing a new major.
